backend/src/main/java/com/tourneynizer/tourneynizer/model/TournamentStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.tourneynizer.tourneynizer.model;
+
+public enum TournamentStatus {
+    CREATED, STARTED, FINISHED;
+}
```

Comment:
Good idea, but we could also keep track of this through the database values (I would add an actualStartTime and change the old startTime to plannedStartTime):
* Created - Actual startTime is null, but planned startTime has a value
* Started - Actual startTime has a value, but endTime is null
* Finished - endTime has a value

Reply:
I think having two different start times is a little too complex for right now.